calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/board_detectors/chessboard_detector.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, **kwargs):
         self.resized_detection = Parameter(bool, value=True, min_value=False, max_value=True)
         self.resized_max_resolution = Parameter(int, value=1000, min_value=500, max_value=3000)
         self.sub_pixel_refinement = Parameter(bool, value=True, min_value=False, max_value=True)
+        super().set_parameters(**kwargs["cfg"])
```

Comment:
I think super is not warranted here

Reply:
I'm not exactly sure if this is the correct answer, but I noticed that the other classes were using the next structure for example:
```
class DataCollector(ParameterizedClass):
    self.set_parameters(**cfg)
```
    
I noticed that in this case, the class:

`ChessBoardDetector(BoardDetector): `

inherits from BoardDetector which inherits from class:

`BoardDetector(ParameterizedClass, QObject):` 

which has the method set_parameters.
 
That is the main reason why I think using super() should be fine, but if that is not the case, I can receive your opinion and modify it accordingly.

Reply:
f2ad5cce4cffc72284d0b575020c40851475872e

Reply:
As explained in person, since we are using polymorphism, super is not warranted.
In addition, it seems the commit you pasted does not address it

Reply:
[ec063e9](https://github.com/tier4/CalibrationTools/pull/220/commits/ec063e916576efd4d43021693ba83ffc22ae1910)